jps-wrapper/src/main/kotlin/jps/wrapper/main.kt

```diff
@@ -58,6 +60,7 @@ private fun runBuild(model: JpsModel) {
             emptyList(),
             Properties.includeTests.toBoolean(),
             { msg ->
+                if (withProgress && msg is ProgressMessage) print("[Progress = ${msg.done}]:")
                 println(msg)
                 if (msg.kind == BuildMessage.Kind.ERROR || msg.kind == BuildMessage.Kind.INTERNAL_BUILDER_ERROR) {
```

Comment:
ProgressMessage is always `BuildMessage.Kind.Progress`. Please create a dedicated branch for progress messages with explicit output constructing.